### DISCUSSION TITLE
Cosmology Slides: asking for reviews

### DISCUSSION THREAD

**Discussion by HanDeBruijn** [a-cosmology-group/acg#189] [Cosmology Models]
**[Cosmology Slides](https://www.astro.rug.nl/~saleem/courses/CosmologySlides.pdf)** are a presentation for students at the RijksUniversiteit Groningen (RUG) in the Netherlands.
One of the slides has been [shown](https://github.com/orgs/a-cosmology-group/discussions/188#discussioncomment-7645966) already in [thread 188](https://github.com/orgs/a-cosmology-group/discussions/188) by @mikehelland.
I could have plugged the rest of the subject in there but that would be at the risk of becoming highly off-topic.
The slides are typical for contemporary cosmology and I think they give a good overview of the state of the **art**.
(If one would be tempted to call it in the latter way)
Therefore I've decided to open a separate thread, hoping for some sensible feedback.

As an appetizer, take a look at this:

![BB](https://github.com/a-cosmology-group/acg/assets/135433222/fd1df2f9-3dd0-4a78-921a-47822db6c37d)
An _incredibly_ detailed **[Chronology of the universe](https://en.wikipedia.org/wiki/Chronology_of_the_universe)** is found at Wikipedia.

**Comment by HanDeBruijn** [comment]
In this comment I want to talk about about **wrong mathematics** that seems to go unnoticed by an audience of students.
Worse. It even goes unnoticed by cosmologists doing the presentation. Take slide 3 - **Olbers' Paradox** -  as an example.

$$
dJ = \frac{nL}{4\pi r^2}r^2 dr = \frac{nL}{4\pi}
$$

_Hence_, they say

$$
J = \int_0^\infty dJ = \int_0^\infty \frac{nL}{4\pi}dr
$$

Whatever the meaning of the other symbols is, $r$ is the radius of a sphere.
Debunking the above shouldn't be difficult for the mathematically inclined among ACG members. Or is it?
Anyway, here is a solution:

$$
dJ = \frac{nL}{4\pi r^2} {\bf \Large 4\pi} r^2 dr =  nL.dr
$$

The final integral becomes infinite. Have people on earth gone blind now? No!
We are saved by the Cosmological Redshift, explained by Variable Mass or Tired Light or whatever, resulting in an _exponential decay_ of the Watts received in our eyes.

$$
J = \int_0^\infty e^{-H_0/c.r} nL.dr = nL\left(\frac{c}{H_0}\right)
$$

For completeness: $n=$ number of stars / volume, $L=$ mean luminosity / star, $c=$ lightspeed, $H_0=$ Hubble parameter.
Olbers' Paradox has disappeared herewith.
Caveat: there is _optical shielding_, as opposed to NO gravitational shielding. A star behind another star cannot be seen. The power / area $J$ thus calculated is somewhat too high.

### Update

Question is: can we shut up and calculate? Let's try to make a crude estimate.
$n=$ (number of stars) / volume = (mean mass density) / (mass of the sun) =

$$
\rho_0 = \frac{H_0^2}{w.8\pi.G} \quad ; \quad M_{sun} = 3955\times 10^{30}\mbox{ }kg
$$

where

$$
w = \frac{2-e^{-W}(W^2+2W+2)}{W}  \quad \mbox{and} \quad W = 3.383634283
$$

Google says: _The luminosity of the Sun is about_ $3.86 \times 10^{26}$ _watts. This is the total power radiated out into space by the Sun._
At last, the [Stefan-Boltzmann law](https://en.wikipedia.org/wiki/Stefan%E2%80%93Boltzmann_law) is, for the _total energy radiated per unit surface area per unit time_, which is also the total energy _received_ by a test particle per unit surface area per unit time $=\sigma\Theta^4$  with $\Theta=$ absolute temperature.
Now we have sufficient ingredients I think.
```
Mpc := 3.08567758*10^22;
H_0 := 73.4*1000/Mpc;
G := 6.67408*10^(-11);
W := 3.383634283;
w := (2-exp(-W)*(W^2+2*W+2))/W;
rho_0 := H_0^2/(w*8*Pi*G);
sun := 3955*10^(30);
n := rho_0/sun;
L := 3.86*10^(26);
c := 299792458;
watt := n*L*c/H_0;
sigma := 5.670374419*10^(-8);
Theta := evalf((watt/sigma)^(1/4));

                         Theta := 1.171582089
```
Hmm. Could refinement of the above possibly lead to another theory that explains the Cosmic Background Radiation ??

**Reply by RedshiftDrift** [reply]
That's equivalent to how poeple calculated the CMB temperature in the 1940s - 60s, see sections 6 and 7 in ["History of the 2.7 K Temperature Prior to Penzias and Wilson"](https://www.researchgate.net/publication/2938670_History_of_the_27_K_Temperature_Prior_to_Penzias_and_Wilson)  The main problem is that the average density of stars is not well known.

**Reply by HanDeBruijn** [reply]
Quote from slide 13:
>The Steady State model was the most serious contender for a cosmological theory and by
the middle sixties it was the preferred model for most astronomers. However, the
discovery of the Cosmic Microwave Background radiation, a thermal relic from the early
Universe, which clearly tipped the balance in favor of the Big Bang model.

Okay, we all "know" this. Alas, I cannot reproduce the mathematics - let it be the physics - on **slide 16**. Here are my attempts.
_Assuming no external sources of energy (heating) then one can use the first law of thermodynamics dQ=dE+PdV to obtain dE= -PdV._
_Substituting the two terms in the first law, equation one obtains the following equation:_

$$
\frac{1}{T}\frac{dT}{dt} = - \frac{1}{3V}\frac{dV}{dt}
$$

Let's check. Starting with $E=\alpha.T^4.V$ and $P=\alpha.T^4/3$ we have:

$$
\frac{dE}{dt} = \frac{\partial E}{\partial T}\frac{dT}{dt} + \frac{\partial E}{\partial V}\frac{dV}{dt} = \alpha.3T^3.V\frac{dT}{dt} + \alpha.T^4\frac{dV}{dt}
$$

$$
-P\frac{dV}{dt} = -\frac{\alpha.T^4}{3}\frac{dV}{dt}
$$

Equate and divide everything by $(\alpha.3T^4.V)$ 

$$
\frac{1}{T}\frac{dT}{dt} + \frac{1}{3V}\frac{dV}{dt} = - \frac{1}{9V}\frac{dV}{dt}
$$

Conclusion: that equation on slide 16 can NOT be derived in this way.
It **could** have been derived with $dE=0$, thus assuming that the total energy of the CMB doesn't change.
On slide 25 we find that $\Omega = 1$ and _The total energy of the Universe is zero_. What do they mean by that ??
Whatever, with $V = V_0.a(t)^3$ we get, at last

$$
\frac{\dot{T}}{T} = \frac{1}{T}\frac{dT}{dt} = -\frac{1}{3V}\frac{dV}{dt} = -\frac{1}{3 a(t)^3}3a(t)^2\frac{da}{dt} = - \frac{\dot{a}}{a}
$$

giving indeed $T(t) \sim a(t)^{-1}$ as stated.

$$
\frac{\dot{T}}{T} + \frac{\dot{a}}{a} = \frac{\dot{T}.a+\dot{a}.T}{a.T} = \frac{\dot{(a.T)}}{a.T} = 0 \quad \Longleftrightarrow \quad a.T = C \neq 0 \quad \Longleftrightarrow \quad T=C/a
$$

Product rule for differentiation. Sorry. I don't like to skip over details.

**Comment by bgreenman** [comment]
The Assis ans Neves January 1995 paper on Regener and Nernst work in 1993 describing an alternative model of the CMB is a good stating point in researching he CMB.

**Comment by HanDeBruijn** [comment]
Illustrative video: [Line Element In Spherical Coordinates](https://www.youtube.com/watch?v=YLkRBngPiZs). Quote:
>$l = R\int\sqrt{d\theta^2+\sin^2(\theta)d\phi^2}\mbox{ }$ ----- (3) This is line element on the surface of a sphere

Other relevant references from the internet are: [Geographic coordinate system](https://en.wikipedia.org/wiki/Geographic_coordinate_system), [Spherical Coordinates](https://mathworld.wolfram.com/SphericalCoordinates.html), [Spherical coordinate system](https://en.wikipedia.org/wiki/Spherical_coordinate_system).
Spherical coordinates are defined here together with Cartesian coordinates $(x,y,z)$ according to the physical convention.

$$
\begin{cases}
x = r\sin(\theta)\cos(\phi) \\
y = r\sin(\theta)\sin(\phi) \\
z = r\cos(\theta)
\end{cases}
$$

With accompanying infinitesimals:

$$
\begin{cases}
dx = dr \sin(\theta)\cos(\phi) + r\cos(\theta)\cos(\phi) d\theta - r\sin(\theta)\sin(\phi) d\phi \\
dy = dr \sin(\theta)\sin(\phi) + r\cos(\theta)\sin(\phi) d\theta + r\sin(\theta)\cos(\phi) d\phi \\
dz = dr \cos(\theta) - r\sin(\theta) d\theta
\end{cases}
$$

When putting this in matrix form, preferrably an orthogonal matrix shall be emerging, because that saves us a lot of tedious algebra.
Each column of such a matrix must be perpendicular to each other column (i.e. inner product $=0$) and the length of each column must be $=1$.
After some puzzling we find:

$$
\begin{bmatrix} dx \\
dy \\ 
dz \end{bmatrix} = 
\begin{bmatrix} \sin(\theta)\cos(\phi) & \cos(\theta)\cos(\phi) & -\sin(\phi) \\
                \sin(\theta)\sin(\phi) & \cos(\theta)\sin(\phi) & +\cos(\phi) \\
                \cos(\theta)           & -\sin(\theta)          & 0 \end{bmatrix}
\begin{bmatrix} dr \\
r d\theta \\
r\sin(\theta) d\phi \end{bmatrix}
$$

Then what we have is an orthogonal transformation and immediately so:

$$
dx^2+dy^2+dz^2 = dr^2 + (r d\theta)^2 + (r\sin(\theta) d\phi)^2
$$

On the surface of the sphere with radius $R$ any infinitesimal increment $dr$ is obviously zero.
So the square of the infinitesimal line element is on that surface is, in concordance with the video we started with

$$
ds^2 = R^2\left[d\theta^2 + \sin^2(\theta)d\phi^2\right]
$$

This formula on slide 31 of **[Cosmology](https://www.astro.rug.nl/~saleem/courses/CosmologySlides.pdf)** is somewhat confusing.
![slide31](https://github.com/a-cosmology-group/acg/assets/135433222/50fa63ca-1628-42eb-b28f-c2f07f256e1e)
But we can establish an overall correspondence between the different formulations

$$
\begin{matrix}
\mbox{physical} & \phi & \theta \\
\mbox{geography} & \lambda & \pi/2-\phi \\
\mbox{presentation} & \theta & r/R\\
\end{matrix}
$$

So it turns out to be correct in the end:

$$
ds^2 = R^2\left[d(r/R)^2 + \sin^2(r/R)d\theta^2\right] = dr^2 + R^2\sin^2(r/R)d\theta^2
$$

![samen_OK](https://github.com/a-cosmology-group/acg/assets/135433222/25266b0b-b827-4018-b977-e616f5b58ea5)

**Comment by HanDeBruijn** [comment]
>After the Copernican revolution the idea that we are at the center of the Universe has been
discarded once and again. Pushing this notion further one can deduce that indeed there is
no preferred point in the Universe nor there is a preferred direction. These types of
arguments have lead to a basic assumption in Cosmology which states that the Universe
is homogeneous and isotropic. These two assumptions combined constitute what is known
as the "Cosmological Principle".
Note that the two requirements are fundamentally different and having one does not imply
the other. Here are two examples of distributions that are either homogeneous or isotropic:
![behoud](https://github.com/a-cosmology-group/acg/assets/135433222/2ca5203a-795a-4ca0-a750-ba74aac734fa)

The above pictures on slide 6 show in a lucid way what the difference is between isotropic and homogeneous.
Homogeneous = translation invariant. It doesn't matter at which position you are, as long as you look into the same direction.
Isotropic = rotation invariant. It doesn't matter in what direction you are looking, as long as you are at the same position.

But there is something missing. I thought we have space and time **united** nowadays, as coined up by Minkowski: **space-time**.
Why then is **time invariant** not in the list? Oh yes, I know: it is because of the Big Bang (that never happened according to some).
The origin of time can be chosen at will (for autonomous systems) everywhere in physics, with **cosmology as the only exception**.

Here is an instructive website &amp; video: [What is Energy? Is Energy Conserved?](https://backreaction.blogspot.com/2020/10/what-is-energy-is-energy-conserved.html)
Sabine's presentation is exactly to the point. Indeed, [Emmy Noether proved](https://en.wikipedia.org/wiki/Noether%27s_theorem) a few things:

- invariance for translation in space is intimately related to conservation of linear momentum
- invariance for rotation in space is intimately related to conservation of angular momentum
- last but not least: time-translation invariance is intimately related to conservation of energy

Human technology would be unthinkable without experiments which are not repeatable in the first place.
And it is expected that outcomes must be the same at any spatial orientation and at any place around.
So the conservation laws cannot be questioned for the physics that is (technologically) relevant for us.

Furthermore, there exists (advanced) mathematics supporting the relationship between invariance and conservation laws.
Via the theory of [Lie groups](http://www.alternatievewiskunde.nl/MijnBoek/lie.htm) invariance in space and time is linked to the well known quantum mechanical operators.

Another instructive video at last: [Child genius gets advice from astrophysicist Neil DeGrasse Tyson](https://www.youtube.com/watch?v=gBPumrQlN58). Or is it the other way around?

**Reply by bgreenman** [reply]
Agree

**Comment by HanDeBruijn** [comment]
No further comments feasible. End of (not quite successful) discussion. Reason why?
The **Cosmology Slides** have been taken **offline** and put behind a wall for RUG students only.
